### DISCUSSION TITLE
My selected school has incorrect or incomplete information e.g. domains or campus location

### DISCUSSION THREAD

**Discussion by ghost** [community/community#67447] [GitHub Education]
### Hi everyone,

Hi there,

I am trying to apply for student benefits with my student email and ID. I continue to receive the message as below after uploading my student ID photo with the campus network. My student ID has the information of the university name and expiry date. And I have registered with the correct student email address. Could anyone help explain how to solve this problem? Many thanks in advance!

The message goes:
"The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect email domain, please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.

You appear not to be near any campus location for the school you have selected. Your school-provided academic affiliation document must indicate virtual learning if you are remote.Please enable the application to discover your location from the browser pop-up to provide an accurate location. If you are using a VPN, please disable it when you reapply. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect campus location, please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.
"

Best 
Xiaoxiao

**Comment by nmindz** [comment]
Same issue here.

**Comment by jackdmarquez** [comment]
Same here, did you get any solution?

**Reply by nmindz** [reply]
Still hasn't resolved here. 😞
